upi/openstack/inventory.yaml

```diff
@@ -34,11 +33,16 @@ all:
       # Number of provisioned Compute nodes.
       # 3 is the minimum number for a fully-functional cluster.
       os_compute_nodes_number: 3
-
-      acc_provision_tar: /fullpath/to/tar/file
-      opflex_interface: ens4
-      node_interface: ens3
-      aci_containers_addr_scope: addr-scp-openshiftupi 
-      aci_containers_l3out_vrf: uni/tn-common/ctx-sauto_l3out-1_vrf
-      aci_containers_subnet_pool: subpool-openshiftupi
-      aci_containers_prefix_length: 27
+      aci_cni:
+        acc_provision_tar: /home/noiro/openupi/aci_deployment.yaml.tar.gz
+        network_interfaces:
+          node:
+            name: ens3
+            vrf: uni/tn-common/ctx-sauto_l3out-1_vrf
+            # no default MTU setting for node network
+          opflex: 
+            name: ens4
+            subnet: '16.11.0.0/24'
+            mtu: 1600
```

Comment:
I believe this should be 1500, since the neutron network is being set to 1500 MTU by default.

Reply:
I had set it to another value to test the functionality as 1500 is the default anyway, but yes set it back to 1500 to show a default example.